moebot_bot/util/util.go

```diff
@@ -22,6 +22,11 @@ const (
 	CaseSensitive
 )
 
+type SyncSlicesMap struct {
```

Comment:
I think it would be best to make this specific: `SyncStringSliceMap` since we may want intSlice, floatSlice and so on